client/user/src/mealorder/meal-order-client-app.html

```diff
@@ -31,42 +30,222 @@
+                justify-content: space-around;
+                align-items: center;
+            }
+
+            #day {
+                font-size: 0.7em;
+                font-weight: lighter;
+                text-align: center;
+            }
 
+            #day-of-the-week {
+                font-size: 22px;
+            }
+
+            @media only screen and (max-width: 600px) {
+                .route-title {
+                    display: none;
+                }
+
+                app-header .history-icon {
+                    display: block;
+                }
+            }
+        </style>
         <app-location route="{{route}}"
                       use-hash-as-path></app-location>
 
         <app-route
                 route="{{route}}"
-                pattern="/:page"
+                pattern="/:page/:date"
                 data="{{routeData}}"></app-route>
 
         <app-drawer-layout fullbleed>
+            <app-drawer slot="drawer" align="right" class$="[[_getClassForDrawer(routeData.page)]]">
+                <iron-pages id="app-drawer-iron"
+                            selected="[[routeData.page]]"
+                            attr-for-selected="data-page"
+                            fallback-selection="view404"
+                            role="main">
 
-            <app-drawer slot="drawer" align="right">
-                <app-toolbar>
-                </app-toolbar>
+                    <app-toolbar data-page="menu">
+                        <div class="app-toolbar-header">
+                            <div class="app-toolbar-content">Заказ</div>
+                            <div>
+                                <div id="day">[[selectedDay.dayOfTheWeek]]</div>
+                                <div id="day-of-the-week">[[selectedDay.day]]</div>
+                            </div>
+                        </div>
+                    </app-toolbar>
+
+                    <app-toolbar data-page="history">
+                        <div class="app-toolbar-header">
+                            <div class="app-toolbar-content">История заказов</div>
+                        </div>
+                    </app-toolbar>
+                </iron-pages>
+                <meal-order-drawer order="[[order]]"></meal-order-drawer>
             </app-drawer>
 
             <app-header-layout>
                 <app-header slot="header" reveals effects="waterfall">
                     <app-toolbar>
+                        <div class="app-toolbar-container">
+                            <div id="main-title">Заказ обедов</div>
+                            <iron-pages id="app-header-iron"
+                                        selected="[[routeData.page]]"
+                                        attr-for-selected="data-page"
+                                        fallback-selection="view404"
+                                        role="main">
+
+                                <div data-page="menu">
```

Comment:
Why are these `<div>` wrappers needed?

Reply:
If for one routing two `iron-icons` are needed, where one is without a link.